vanilla-framework/vanilla-framework
# p-cards: Apply fixed padding

## Done

Card padding should not be scaleable. I've added a variable ($spv-intra--regular) to handle cases like this, and I've applied it to all the card flavours.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/card/highlighted/ for example
- Verify that at $multi: 2; nothing changes; at $multi: 1 nothing changes as well. Previously, at $multi: 1 the padding would become .5 rem, which is not useful even in the context of high-density ui like maas.

## Details

[List of links to issues/bugs and cards if needed]

## Screenshots
Before: ($multi: 1)
![image](https://user-images.githubusercontent.com/2741678/42464630-8f3cb48e-83a1-11e8-8c20-c10b72b02721.png)



After: ($multi: 1)
![image](https://user-images.githubusercontent.com/2741678/42464595-72586e58-83a1-11e8-8e9d-56b8c5fa32c5.png)
